qa/suites/krbd/mirror/tasks/compare-mirror-image-alternate-primary.yaml

```diff
@@ -0,0 +1,14 @@
+overrides:
+  install:
+    ceph:
+      extra_system_packages:
+        - pv
+tasks:
+- workunit:
+    clients:
+      cluster1.client.mirror:
+        - rbd/compare_mirror_image_alternate_primary.sh
+    env:
+      RBD_DEVICE_TYPE: 'krbd'
+      RBD_MIRROR_USE_RBD_MIRROR: '1'
```

Comment:
Not sure which setting?  These are environment variables that need to be passed to this particular script -- I don't think it would make sense (or even work) anywhere else.

Reply:
I was referring to `RBD_MIRROR_USE_RBD_MIRROR` .  You're right, it's specific to `rbd_mirror_helpers.sh` script.